examples/webxr_vr_sandbox.html

```diff
@@ -193,6 +195,21 @@
 				mesh.scale.setScalar( 2 );
 				group.add( mesh );
 
+
+				// Add stats.js
+				stats = new Stats();
+				stats.dom.style.width = '80px';
+				stats.dom.style.height = '48px';
```

Comment:
How come this is needed?

Reply:
yeah, looks like these 2 lines aren't needed at all to work...